### DISCUSSION TITLE
FeatureRequest: java.lang.Long

### DISCUSSION THREAD

**Discussion by kibae** [mike-lischke/java2typescript#8] [General]
Hello.

I would like to request support for the java.lang.Long type. I tried to develop it by referring to Integer.ts file and [the source of the JDK](http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/f92ab6dbbff8/src/share/classes/java/lang/Long.java), but I am having difficulty in the bit related part.
java.lang.Long is a commonly used type. I hope the project can support this.
Thank you for developing such a great project. 🙏

**Comment by mike-lischke** [comment]
Shouldn't be a big problem to write. Can you move this over to the Issues list, so I can schedule it?

**Reply by kibae** [reply]
#9 I posted as an issue :)
